src/main/java/com/fundation/webservice/model/CriteriaAudio.java

```diff
@@ -17,48 +17,60 @@
  */
 public class CriteriaAudio extends CriteriaConvert {
     private String newFormat;
-    private String aCodec;
-    private int aBit;
-    private int aChannel;
-    private int aRate;
+    private String audioCodec;
+    private int audioBit;
+    private int audioChannel;
+    private int audioRate;
 
+    /**
+     *
+     * Method getNewFormat receive the new format audio.
+     */
     public String getNewFormat() {
         return newFormat;
     }
 
+    /**
+     *
+     * Method setNewFormat establish the new format audio that is define in getNewFormat
+     */
```

Comment:
Single line comment please.

Reply:
it is the name of the class for emphasizing the comment